querybook/webapp/redux/scheduledDataDoc/action.ts

```diff
@@ -25,7 +25,11 @@ export function getScheduledDocs({
             envId,
             limit: pageSize,
             offset: page * pageSize,
-            filters,
+            filters: {
+                ...filters,
+                lists: filters.lists?.map((l) => l.value),
```

Comment:
its not the job of getScheduledDocs to process what filters look like because `Select` is used